docs/docker.rst

```diff
@@ -0,0 +1,74 @@
+Using mBuild with Docker
+========================
+
+As much of scientific software development happens in unix platforms, to avoid the quirks of development dependent on system you use, a recommended way is to use docker or other containerization technologies. This section is a how to guide on using mBuild with docker.
+
+Prerequisites
+-------------
+A docker installation in your machine. Follow this `link <https://docs.docker.com/get-docker/>`_ to get a docker installation working on your machine.
+
```

Comment:
Somewhere in this document can we link to a basic docker tutorial/intro? As someone who has never used docker before everything is a bit cryptic. Here is one website I found useful: https://docker-curriculum.com/, but I am open to other suggestions. I just think it would be beneficial to have something we can point people towards for a top level intro to docker.